module/data/item/templates/physical-item.mjs

```diff
@@ -0,0 +1,66 @@
+/**
+ * Data model template with information on physical items.
+ *
+ * @property {number} quantity     Number of items in a stack.
+ * @property {number} weight       Item's weight in pounds or kilograms (depending on system setting).
+ * @property {number} price        Item's cost in GP.
+ * @property {number} attunement   Attunement information as defined in `DND5E.attunementTypes`.
+ * @property {boolean} equipped    Is this item equipped on its owning actor.
+ * @property {string} rarity       Item rarity as defined in `DND5E.itemRarity`.
+ * @property {boolean} identified  Has this item been identified?
+ * @mixin
+ */
+export default class PhysicalItemTemplate extends foundry.abstract.DataModel {
+  static defineSchema() {
+    return {
+      quantity: new foundry.data.fields.NumberField({
+        required: true, nullable: false, integer: true, positive: true, initial: 1, label: "DND5E.Quantity"
```

Comment:
`positive: true` here is perhaps too strict. I know there are some users who like to keep things at `0` quantity so they don't forget to restock them and can do so more easily.